database/migrations/2022_01_05_151733_add_year_to_global_clusters.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddYearToGlobalClusters extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('clusters', function (Blueprint $table) {
+            $table->year('year')->nullable()->index();
+
+            $table->index(['year', 'zoom']);
```

Comment:
Do we need to index year again here? Is it doing an index combination of year + zoom?

Reply:
Yes, it's doing a composite index. In our queries, we will always be doing queries with both year and zoom, even when the year is null.